frontend/src/MuiForms/Dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import { Autocomplete, TextField, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { Fragment, useMemo } from 'react'
+
+interface DropDownProps {
+  label?: string
+  required?: boolean
+  disabled?: boolean
+  readOnly?: boolean
+  formContext?: any
+  value: string
+  onChange: (newValue: string) => void
+  InputProps?: any
+  options: any
+}
+
+export default function DropDown(props: DropDownProps) {
+  const { onChange, value, label, formContext, options } = props
+
+  const theme = useTheme()
+
+  const handleChange = (_event: React.SyntheticEvent<Element, Event>, newValue: any) => {
+    if (newValue) {
+      onChange(newValue.value)
+    }
+  }
+
+  const disabledWebkitTextFillColor = useMemo(() => {
+    if (value) {
+      return theme.palette.mode === 'light' ? theme.palette.common.black : theme.palette.common.white
+    } else {
+      return theme.palette.customTextInput.main
+    }
+  }, [theme, value])
+
+  const dropDownOptions = useMemo(() => {
+    return options.enumOptions ? options.enumOptions.map((option) => option.value) : []
+  }, [options])
+
+  return (
+    <Fragment key={label}>
+      <Typography fontWeight='bold'>{label}</Typography>
+      {formContext.editMode && (
+        <Autocomplete
+          size='small'
+          options={dropDownOptions || []}
+          sx={{
+            input: {
+              color: theme.palette.mode === 'light' ? theme.palette.common.black : theme.palette.common.white,
+            },
+            label: {
+              WebkitTextFillColor:
+                theme.palette.mode === 'light' ? theme.palette.common.black : theme.palette.common.white,
+            },
+            '& .MuiInputBase-input.Mui-disabled': {
+              WebkitTextFillColor: disabledWebkitTextFillColor,
+            },
+          }}
+          onChange={handleChange}
+          value={value || (!formContext.editMode ? 'Unanswered' : '')}
```

Comment:
Setting the `value` to `'Unanswered'` will cause a react error as it's not one of the options. You should instead set that as a `placeholder` on the `renderInput` `TextField` below